ctp2_code/ui/aui_common/aui_mouse.cpp

```diff
@@ -110,55 +91,44 @@ AUI_ERRCODE aui_Mouse::InitCommon( void )
 	} else {
 		m_sensitivity = 1.0;
 	}
-	m_privateMix = NULL;
-	m_pickup = NULL;
-	m_prevPickup = NULL;
 	m_suspendCount = 0;
 	m_showCount = 0;
 	m_reset = TRUE;
-#ifndef __AUI_USE_SDL__
+	m_flags = 0;
+
+#ifdef __AUI_USE_DIRECTX__
+	m_privateMix = NULL;
+	m_pickup = NULL;
+	m_prevPickup = NULL;
+
 	m_thread = NULL;
 	m_threadId = 0;
 	m_threadEvent = NULL;
 	m_terminateEvent = NULL;
 	m_suspendEvent = NULL;
 	m_resumeEvent = NULL;
 	m_replyEvent = NULL;
-#endif
-	m_flags = 0;
+#endif // __AUI_USE_DIRECTX__
 
 	memset( m_cursors, 0, sizeof( m_cursors ) );
 	memset( &m_data, 0, sizeof( m_data ) );
 	memset( m_inputs, 0, sizeof( m_inputs ) );
 
 	SetClip( NULL );
+
 #ifdef __AUI_USE_DIRECTX__
 	GetCursorPos( &m_data.position );
-#elif defined(__AUI_USE_SDL__)
-        int x = 0, y = 0;
-        Uint8 state = SDL_GetMouseState(&x, &y);
```

Comment:
The initialization of those members have been moved to the constructor of aui_SDLMouse.